modules/reference/pages/command-line-tools.adoc

```diff
@@ -695,6 +695,68 @@ Show help message and exit.
 
 
 
+[[spacewalk-repo-sync]]
+== Synchronize Software Repositories into {productname} Channels (spacewalk-repo-sync)
+
+[command]``spacewalk-repo-sync`` is an important tool on {productname}, which works most of the time in the background.
+With [command]``spacewalk-repo-sync`` you synchronize software repositories into {productname} channels.
+
+But for debugging or solving smaller issues it is useful to run it directly with special parameters.
```

Comment:
```suggestion
This can be useful for debugging or for solving some synchronization problems.
```

Reply:
Notes on edits to these three lines:
* The first line should explain what the tool does
* The second line then goes on to explain the circumstances in which you would use it.

It's the same principle as sentence structure, but on a higher level: the first part is the most important, the second part is the lens.